Dockerfile

```diff
@@ -12,7 +10,8 @@ LABEL com.github.actions.description="Wraps the vsce CLI to be used in GitHub Ac
 LABEL com.github.actions.icon="package"
 LABEL com.github.actions.color="green"
 
+RUN apk add --no-cache bash
+RUN apk add --update nodejs npm 
 RUN npm install -g vsce@$npm_version
 
-COPY "entrypoint.sh" "/entrypoint.sh"
-ENTRYPOINT ["/entrypoint.sh"]
+COPY entrypoint.sh /entrypoint.sh
```

Comment:
I'm not sure I understand the removal of the ENTRYPOINT directive. Can you explain why it's needed?

Reply:
Accidentally deleted. Restored it.